### PR TITLE
markup + styles

### PR DESCRIPTION
**Pull request by agessler**
icon-widget

### CODE REVIEW

File: src/Components/IconEditorTab/IconEditorTab.tsx [hunk 1]
```diff
@@ -24,14 +24,47 @@ export const IconEditorTab: React.FC<IconEditorTabProps> = ({ widget }) => {
...
+                <i
+                  className={`bs-icon ${currentIcon}`}
+                  style={{
+                    color: color,
+                  }}
+                ></i>
+                <span className="button-del" title="Clear icon">
+                  －
+                </span>
+              </div>
```
Comment (apepper) [inline src/Components/IconEditorTab/IconEditorTab.tsx:39]: The minus in the red "clear" button seems to be misaligned (a little bit too high and not in the center):

Screenshot 2024-09-12 at 14 34 10

File: src/Components/IconEditorTab/IconEditorTab.scss [hunk 2]
```diff
@@ -136,27 +137,72 @@
...
+      .bs-icon {
+        font-size: 14px;
+      }
     }
   }
 }
+
+// -- Layout -- //
+
+.icon-grid {
```
Comment (apepper) [inline src/Components/IconEditorTab/IconEditorTab.scss:188]: I don't know, where this is coming from, but the icon editor looks cut of:

Screenshot 2024-09-12 at 14 34 24

I tried it with Brave on [the preview deployment](https://edit.scrivito.com/d0a154d76edf2a7bd991fc658e700a1d~https://agessler-poc-icon-editor.scrivito-portal-app.pages.dev/de).

File: src/Components/IconEditorTab/IconEditorTab.scss [hunk 3]
```diff
@@ -50,41 +59,38 @@
...
+          0 0 0 2px #507ab5 inset;
         .bs-icon {
-          font-size: 23px;
-        }
-        .text-muted {
-          color: #bbe2d5;
+          font-size: 14px;
         }
       }
       &.active {
```
Comment (apepper) [inline src/Components/IconEditorTab/IconEditorTab.scss:84]: Could you convert the `<a href="#"` tags in `AllIcons.tsx`, `IconSearch.tsx` and `SingleIcon.tsx` to `<button` tags? I assume, that would also need to adjust some stylings then.

File: src/Components/IconEditorTab/IconEditorTab.scss [hunk 4]
```diff
@@ -136,27 +137,72 @@
...
   }
+
+  .neoletter-form-icon-editor-tab .icon-editor-preview .button-del {
+    box-shadow: 0 0 4px 1px rgba(0, 0, 0, 0.8);
+  }
+
   #neoletter-icon-search .bs-icon {
     color: #aaa;
   }
   #neoletter-icon-search #search-input {
```
Comment (apepper) [inline src/Components/IconEditorTab/IconEditorTab.scss:148]: Could you remove `tabIndex={1}` from `<input id="search-input"`? At least my eslint [is warning about it](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/65c9338c62c558d3c1c2dbf5ecc55cf04dbfe80c/docs/rules/tabindex-no-positive.md).

**Review by apepper — CHANGES_REQUESTED** [review]
FEEDBACK ([c319e65](https://github.com/Scrivito/scrivito-portal-app/pull/603/commits/c319e653ed68b96ef63f081b9538990d54d22f7e))

Nice start 🚀. Please see my comments for further improvements.

File: src/Components/IconEditorTab/IconEditorTab.scss (hunk 3 above)
Comment (agessler) [reply src/Components/IconEditorTab/IconEditorTab.scss:98]: FIXED in #[3e3b10a]

File: src/Components/IconEditorTab/IconEditorTab.scss (hunk 2 above)
Comment (agessler) [reply src/Components/IconEditorTab/IconEditorTab.scss:206]: FIXED in #[3e3b10a]

File: src/Components/IconEditorTab/IconEditorTab.tsx (hunk 1 above)
Comment (agessler) [reply src/Components/IconEditorTab/IconEditorTab.tsx]: FIXED in #[3e3b10a]

File: src/Components/IconEditorTab/IconEditorTab.scss (hunk 2 above)
Comment (apepper) [reply src/Components/IconEditorTab/IconEditorTab.scss:206]: It still looks broken

In Brave: 
Screenshot 2024-09-13 at 11 42 50

In Firefox:
Screenshot 2024-09-13 at 11 42 40

Comment (apepper) [reply src/Components/IconEditorTab/IconEditorTab.scss:206]: Strange: on localhost I can't reproduce the issue.

Comment (apepper) [reply src/Components/IconEditorTab/IconEditorTab.scss:206]: So on http://localhost:8080 it is working, but not on https://agessler-poc-icon-editor.scrivito-portal-app.pages.dev/de . Any idea why?

Comment (agessler) [reply src/Components/IconEditorTab/IconEditorTab.scss:206]: Looks like some assets are not loaded or old cache files.

**Comment by apepper** [comment]
Thank you for your work. It will be incorporated independently.

**Pull request closed without merging**